tests/api/patron_types/test_patron_types_permissions.py

```diff
@@ -87,42 +90,84 @@ def test_patron_types_permissions_api(client, librarian_martigny,
+        'search', record=None).can()
+    assert not PatronTypePermissionPolicy(
+        'read', record=patron_type_adults_martigny).can()
+    assert not PatronTypePermissionPolicy(
+        'create', record={}).can()
+    assert not PatronTypePermissionPolicy(
+        'create', record=patron_type_adults_martigny).can()
+    assert not PatronTypePermissionPolicy(
+        'update', record=patron_type_adults_martigny).can()
+    assert not PatronTypePermissionPolicy(
+        'delete', record=patron_type_adults_martigny).can()
 
-    # As non Librarian
-    ptty = patron_type_adults_martigny
-    assert not PatronTypePermission.list(None, ptty)
-    assert not PatronTypePermission.read(None, ptty)
-    assert not PatronTypePermission.create(None, ptty)
-    assert not PatronTypePermission.update(None, ptty)
-    assert not PatronTypePermission.delete(None, ptty)
+    # Patron
+    login_user(patron_martigny.user)
+    assert not PatronTypePermissionPolicy(
+        'search', record=None).can()
+    assert not PatronTypePermissionPolicy(
+        'read', record=patron_type_adults_martigny).can()
+    assert not PatronTypePermissionPolicy(
+        'create', record={}).can()
+    assert not PatronTypePermissionPolicy(
+        'create', record=patron_type_adults_martigny).can()
+    assert not PatronTypePermissionPolicy(
+        'update', record=patron_type_adults_martigny).can()
+    assert not PatronTypePermissionPolicy(
+        'delete', record=patron_type_adults_martigny).can()
 
     # As Librarian
-    with mock.patch(
-        'rero_ils.modules.patron_types.permissions.current_librarian',
-        librarian_martigny
-    ):
-        assert PatronTypePermission.list(None, ptty)
-        assert PatronTypePermission.read(None, ptty)
-        assert not PatronTypePermission.create(None, ptty)
-        assert not PatronTypePermission.update(None, ptty)
-        assert not PatronTypePermission.delete(None, ptty)
+    login_user(librarian_martigny.user)
+    assert PatronTypePermissionPolicy(
+        'search', record=None).can()
+    assert PatronTypePermissionPolicy(
+        'read', record=patron_type_adults_martigny).can()
+    assert not PatronTypePermissionPolicy(
+        'read', record=patron_type_youngsters_sion).can()
+    assert not PatronTypePermissionPolicy(
+        'create', record={}).can()
+    assert not PatronTypePermissionPolicy(
+        'create', record=patron_type_adults_martigny).can()
+    assert not PatronTypePermissionPolicy(
+        'create', record=patron_type_youngsters_sion).can()
+    assert not PatronTypePermissionPolicy(
+        'update', record=patron_type_adults_martigny).can()
+    assert not PatronTypePermissionPolicy(
+        'update', record=patron_type_youngsters_sion).can()
+    assert not PatronTypePermissionPolicy(
+        'delete', record=patron_type_adults_martigny).can()
+    assert not PatronTypePermissionPolicy(
+        'delete', record=patron_type_youngsters_sion).can()
```

Comment:
ergonomy consideration : maybe i will group all tests for the same organisation (Martigny), followed by tests for other (Sion) organisation.